sklearn/ensemble/tests/test_base.py

```diff
@@ -24,8 +24,8 @@
 
 def test_base():
     # Check BaseEnsemble methods.
-    ensemble = BaggingClassifier(base_estimator=Perceptron(random_state=None),
-                                 n_estimators=3)
+    ensemble = BaggingClassifier(
+        base_estimator=Perceptron(tol=1e-1, random_state=None), n_estimators=3)
```

Comment:
If these will work with default tolerance settings before and after the deprecation, I think it would be best to leave it unspecified.